example/README.md

```diff
@@ -37,3 +37,38 @@
    - `./reset-all.sh`
 5. Start the test app
    - `yarn start:example`
+
+## Running (Windows)
+
+1. Open Powershell and navigate to dm-core-packages/example.
+2. Make sure docker engine is running, then run these commands
+   - `yarn install`
```

Comment:
yarn install should be run from dm-core-packages folder

Reply:
Yarn install is now from the correct folder, thanks for spotting that. 

Does not need yarn build:dm-core and dm-core-plugins. 
Was not able to get the commands to run without errors, but it works nonetheless without building. And Andrea said it should work without building the packages locally.